public/components/trace_analytics/requests/queries/services_queries.ts

```diff
@@ -204,7 +230,117 @@ export const getServiceMetricsQuery = (DSL: any, serviceNames: string[], map: Se
           )
         )
       : [].concat(...Object.keys(map).map((service) => getServiceMapTargetResources(map, service)));
-  const query: any = {
+  const jaegerQuery: any = {
+    size: 0,
+    query: {
+      bool: {
+        must: [],
+        should: [],
+        must_not: [],
+        filter: [
+          {
+            terms: {
+              "process.serviceName": serviceNames,
+            },
+          },
+          {
+            bool: {
+              should: [
+                {
+                  bool: {
+                    filter: [
+                      {
+                        bool: {
+                          must_not: {
+                            term: {
+                              references: {
+                                value: [],
+                              },
+                            },
+                          },
+                        },
+                      },
+                      // {
+                      //   terms: {
+                      //     name: targetResource,
+                      //   },
+                      // },
```

Comment:
Let's remove the comments as well

Reply:
Fixed in [45cdc1e](https://github.com/opensearch-project/dashboards-observability/pull/150/commits/45cdc1e63436848316f6aa1ab5f6f352fcfa9a11)